protocol/statetracker/event_tracker.go

```diff
@@ -74,6 +84,12 @@ func (et *EventTracker) updateBlockResults(latestBlock int64) (err error) {
 	return nil
 }
 
+func (et *EventTracker) getLatestBlockTime() time.Time {
```

Comment:
event tracker tracks events not blocks and their time, remove it form here you can use chaintracker time